local_install.R

```diff
@@ -1,8 +1,17 @@
+# Installs the custom-made packages in this repository
+
 library(devtools)
   
+# To run if operating in the container
 initial.options <- commandArgs(trailingOnly = FALSE)
 file.arg.name <- "--file="
-script.name <- sub(file.arg.name, "", initial.options[grep(file.arg.name, initial.options)])
-
-pkg.dir <- paste0(dirname(script.name), "/R/pkgs/")
+script.name <- sub(file.arg.name, "", initial.options[grep(file.arg.name, initial.options)]) # get the name of this file, by looking for the option "--file" in the arguments that were used to start this R instance and getting the term that comes after
```

Comment:
Generally to make code review easier, we prefer comments on their own lines.  Not an issue for this PR